pkg/utils/secrets/certificates.go

```diff
@@ -178,11 +178,11 @@ func (s *CertificateSecretConfig) generateCertificateTemplate() *x509.Certificat
 
 		template = &x509.Certificate{
 			BasicConstraintsValid: true,
-			IsCA:         isCA,
-			SerialNumber: serialNumber,
-			NotBefore:    now,
-			NotAfter:     now.AddDate(10, 0, 0), // + 10 years
-			KeyUsage:     x509.KeyUsageDigitalSignature | x509.KeyUsageKeyEncipherment,
+			IsCA:                  isCA,
```

Comment:
`make verify` results in a dirty git repository:

```
$ git s
On branch pull-226
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   pkg/client/kubernetes/base/base.go
	modified:   pkg/utils/secrets/certificates.go
```

Please fix the formatting.